### ISSUE TITLE
Add onLoad event

### ISSUE THREAD

**Issue by himadrinath** [mervick/emojionearea#152]
i want to add other buttons with my emoji emojionearea-button inside emojionearea 

like this 

![text](https://user-images.githubusercontent.com/12247490/29811312-a675049a-8cc0-11e7-9e8d-59e18f11af91.png)

**Comment by mervick** [comment]
Sorry, currently there is no events for this, I will add new events in new version,
now you can use `setTimeout()` or `setInterval()` function for checking `input` of emojionearea for being ":hidden" and after that add yours buttons,
something like 
```js
$("#emojionearea").emojionearea();
var ready = function(next) {
  var timer = setInterval(function() {
    if ($("#emojionearea").is(":hidden")) {
      clearInterval(timer);
      next();
    }
  }, 200);
};

ready(function() {
   // add your buttons to emojionearea here
});
```

**Comment by himadrinath** [comment]
@mervick  thanks 👍

**Comment by carpii** [comment]
@mervick I'm trying to find ways I could contribute to emojionearea

How do you envision this event working?

Maybe an event being fired when emojionearea is about to add the smiley/close button 
and the event contains the markup which is about to be added?

The event handler can return the full markup which will then be appended?

Or some other idea?

**Comment by mervick** [comment]
this issue id duplicate of https://github.com/mervick/emojionearea/issues/78

**Comment by mervick** [comment]
@carpii if you have a time you can add onLoad event it must be triggered after init function done, myabe here 
https://github.com/mervick/emojionearea/blob/master/src/function/init.js#L544

**Comment by carpii** [comment]
@mervick, thanks I will have a play around tomorrow and see how well it works

**Comment by mervick** [comment]
@carpii I want to completely rewrite this plugin but I don't have a time to do this

**Comment by carpii** [comment]
@mervick I understand. Even in its present state though, its the best I've seen and still helping a lot of people :)

**Comment by mervick** [comment]
@carpii I need help with documentation, if you can do it I would be very grateful

**Comment by carpii** [comment]
@mervick Ok I'll try to help with that too, but I dont have an in-depth understanding of it yet, only the bits I've used in my project. I'll submit some doc changes over the weekend, and then you can decide if theyre worth committing

**Comment by mervick** [comment]
ok

**Comment by carpii** [comment]
@mervick does this new onload event need to use the internal trigger/localstorage mechanism?

Since it will only fire once, from what I can see its just a case of calling trigger("onload")

**Comment by mervick** [comment]
it will used for users and fire only once per instance
